native/src/driver_app.erl

```diff
@@ -0,0 +1,53 @@
+-module(driver_app).
+
+-behaviour(application).
+
+%% Application callbacks
+-export([start/2, stop/1,read_input/0]).
+
+
+
+start(_StartType, _StartArgs) ->
+    read_input(),
+    driver_sup:start_link().
+
+stop(_State) ->
+    ok.
+
+%%====================================================================
```

Comment:
Unless this is some kind of convention in the Erlang community, do not use these kind of fat section separators.